assets/i18n/en.json

```diff
@@ -711,9 +737,15 @@
     "I18N_LEARNER_GROUP_SYLLABUS_ITEM_NOT_STARTED_YET": "Not started yet",
     "I18N_LEARNER_GROUP_SYLLABUS_TAB": "Syllabus",
     "I18N_LEARNER_GROUP_VIEW_DETAILS": "View Details",
+    "I18N_LEARNER_GROUP_VIEW_OVERVIEW_SUMMARY_TITLE": "Your overall progress through syllabus",
+    "I18N_LEARNER_GROUP_VIEW_PREFERENCES": "GROUP PREFERENCES",
     "I18N_LEARNER_GROUP_WITHDRAW_INVITE": "Withdraw",
     "I18N_LEARNT_TOPIC": "Learnt <[topicName]>",
     "I18N_LEARN_TOPIC": "Learn <[topicName]>",
+    "I18N_LEAVE_LEARNER_GROUP": "LEAVE GROUP",
+    "I18N_LEAVE_LEARNER_GROUP_MODAL_BODY": "Are you sure you want to leave <[groupName]> learner group?",
```

Comment:
In some places `groupName` is in quotes, in some it is bolded and here it is without any stuff around, please unify it.

Reply:
Have made it generalised except for 1 case where this was used in a different scenario. Thanks!

Reply:
Hmm, why is that a different scenario?

Reply:
This whole text is itself bold where it is used in html, so to differentiate learner group name, quotes are added there instead of making it bold.

Reply:
Hmm, okay, makes sense.